com.getstation.Station.yaml

```diff
@@ -0,0 +1,50 @@
+id: com.getstation.Station
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: station
+separate-locales: false
+rename-desktop-file: station-desktop-app.desktop
+rename-icon: station-desktop-app
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --env=ELECTRON_TRASH=gio
+modules:
+  - name: station-desktop
+    buildsystem: simple
+    build-commands:
+      - ar x Station-*.deb
+      - rm -f Station-*.deb
+      - tar xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
```

Comment:
no need to rm -rf here, it's in build directory it's not being installed.

Reply:
is it a problem?

Reply:
it's not necessary therefor it shouldn't be here.